ci/travis/install-bazel.sh

```diff
@@ -50,21 +50,15 @@ else
   target="./install.sh"
   curl -s -L -R -o "${target}" "https://github.com/bazelbuild/bazel/releases/download/${version}/bazel-${version}-installer-${platform}-${achitecture}.sh"
   chmod +x "${target}"
-  "${target}" --user > /dev/null
+  if [ "${TRAVIS-}" = true ] || [ -n "${GITHUB_WORKFLOW-}" ]; then
+    sudo "${target}" > /dev/null  # system-wide install for CI
+    command -V bazel 1>&2
```

Comment:
what does this do?

Reply:
It prints something like `bazel is /home/username/bin/bazel` to stderr, that's all.